cmd/dependencycheck/dependencycheck.go

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Checks for restricted dependencies in go packages.
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"regexp"
+)
+
+var (
+	exclude  = flag.String("exclude", `a^`, "skip packages pattern")
```

Comment:
we should probably specify that this is a regex. does this default make sense?

Reply:
Agreed that we should specify. The default here should never match.

Reply:
However, we could just set it to empty string and if exclude is empty string we can return immediately (no reason to check if we don't care what is matched)

Reply:
yeah, leave empty and ignore if unset